src/api/flash/AVRGirlFlasher.js

```diff
@@ -15,90 +15,28 @@
  */
 
 import { logger } from "@api/log";
-import { getStaticPath } from "@renderer/config";
 import AvrGirl from "avrgirl-arduino";
-import path from "path";
-import { spawn } from "child_process";
+const { SerialPort } = require("serialport");
 
-import { delay, toStep } from "./utils";
-
-const AvrDude = async (_, port, filename, options) => {
-  const callback = options
-    ? options.callback
-    : function () {
-        return;
-      };
-  const device = options.device;
-  const timeout = 1000 * 60 * 5;
-
-  const runCommand = async (args) => {
-    await toStep(callback)("flash");
-    return new Promise((resolve, reject) => {
-      const avrdude = path.join(
-        getStaticPath(),
-        "avrdude",
-        process.platform,
-        "avrdude"
-      );
-      logger("flash").debug("running avrdude", { args: args });
-      const child = spawn(avrdude, args);
-      child.stdout.on("data", (data) => {
-        logger("flash").debug("avrdude:stdout:", { data: data.toString() });
-      });
-      child.stderr.on("data", (data) => {
-        logger("flash").debug("avrdude:stderr:", { data: data.toString() });
-      });
-      const timer = setTimeout(() => {
-        child.kill();
-        logger("flash").debug("avrdude timed out");
-        reject("avrdude timed out");
-      }, timeout);
-      child.on("exit", (code) => {
-        clearTimeout(timer);
-        if (code == 0) {
-          logger("flash").debug("avrdude done");
-          resolve();
-        } else {
-          logger("flash").error("avrdude exited abnormally", {
-            errorCode: code,
-          });
-          reject("avrdude exited abnormally");
-        }
-      });
-    });
-  };
+import { toStep } from "./utils";
 
+const rebootToNormal = async (port, _) => {
+  logger("flash").debug("rebooting to normal mode");
   try {
-    await port.close();
-  } catch (_) {
-    /* ignore the error */
+    const serial = new SerialPort({
+      path: port.path,
+      baudRate: 9600,
+    });
+    serial.write("E");
```

Comment:
Can this get documented with comments, possibly even comments pointing to a spec?